Cargo.toml

```diff
@@ -14,8 +14,10 @@ version = "0.1.0"
 [dependencies]
 once_cell = "1.4"
 pin-project-lite = "0.1"
-tokio_02 = {version = "0.2", package = "tokio", features = ["rt-threaded"]}
+tokio_02 = {version = "0.2.22", package = "tokio", features = ["rt-threaded"]}
+tokio_03 = {version = "0.3", package = "tokio"}
+bytes = "0.5.6"
 
 [dev-dependencies]
 hyper = "0.13"
-tokio = {version = "0.3",  features = ["full"]}
+tokio_03 = {version = "0.3", package = "tokio", features = ["full"]}
```

Comment:
The example won't compile will it?

Reply:
Ah thanks. I have to do it like this due to the following error:
```
error: the crate `tokio-compat-02 v0.1.0 (/home/alice/src/tokio-compat-02)` depends on crate `tokio v0.3.0` multiple times with different names
```